src/rbac/rbac.js

```diff
@@ -162,11 +168,15 @@ class RBAC {
 
     const user = this.getUser(userName);
 
-    const userRoleName = user ? user.role : this._defaultRole;
-    const userRole = this.getRole(userRoleName);
-    if (!userRole) return false;
+    const userRoleNames = user ? user.roles : this._defaultRole;
 
-    return this.checkRole(userRoleName, action);
+    for (var i = 0; i < userRoleNames.length; i++) {
```

Comment:
dont use `var` here. use `const` or `let`. also iterating an array should be done with `forEach` as its more readable than a for loop

Reply:
actually instead of `forEach` could use `Array.some()` here pretty effectively.

Reply:
I wrote it this way to short circuit, but using `some` and `every` to break early seems cleaner.